### PR TITLE
Tabbed Restautant page using Javascript

### PR DESCRIPTION
**Pull request by raheebwa**
## Highlights
* [x] Bootsrap used for css styling
* [x] All pages are diffeent modules
* [x] Tabs implemented using only Javascript
* [x] Functional programming used mostly

### CODE REVIEW

File: .eslintrc.js
Comment (shloch) [inline .eslintrc.js]: the config in this file should be same as the one provided by microverse [[details here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json)]

File: package.json
Comment (shloch) [inline package.json]: Necessary run common commands like `build`, `development` should be in `package.json` inside `"scripts"`.  [  [see requirements ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png)]

```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

File: src/modules/home.js
Comment (shloch) [inline src/modules/home.js]: this is creative, i love it 😊😊

**Review by shloch — CHANGES_REQUESTED** [review]
## hello @raheebwa 

AWESOME JOB...i like your product, it's very professional 😊😊
You organise your code very well
Screen Shot 2020-02-14 at 6 53 07 AM

### [changes required]
Your code works well, but let's polish up some things to make the whole repo more professional and some aspects to meet with requirements 😊. Details below :

- below this page, take a look at this conflicting message, please try to debug that
Screen Shot 2020-02-14 at 6 56 34 AM

More about corrections below :
### Waiting for another review 😊

File: src/modules/home.js
Comment (raheebwa) [reply src/modules/home.js]: Thanks @shlock. Will fix the issues and resubmit

**Comment by raheebwa** [comment]
@shloch I was able to work on the request changes. Thanks for your review

**Review by Oghenebrume50 — APPROVED** [review]
### STATUS: APPROVED 

Hi @raheebwa good job on this one it really looks good, :+1:  and kudos for making all suggested changes :slightly_smiling_face:  **This is approved** 

**RAPHAEL NORIODE** :sunglasses:

**Pull request merged**
